bundles/org.openhab.binding.speedtest/src/main/java/org/openhab/binding/speedtest/internal/speedtestConfiguration.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) 2010-2021 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.speedtest.internal;
+
+/**
+ * The {@link speedtestConfiguration} class contains fields mapping thing configuration parameters.
+ *
+ * @author Brian Homeyer - Initial contribution
+ */
+public class speedtestConfiguration {
```

Comment:
```suggestion
public class SpeedtestConfiguration {
```

Classes should start with Capital.

https://www.openhab.org/docs/developer/guidelines.html#java-coding-style

Reply:
done